worker/changestream/eventmultiplexer/eventmultiplexer.go

```diff
@@ -239,6 +257,8 @@ func (e *EventMultiplexer) loop() error {
 			if err := e.dispatchSet(changeSet); err != nil {
```

Comment:
The one metric I want to see, is how long a dispatch take. That's the key to seeing issues.

Reply:
yeah you are right, that's the most interesting metric. Done